src/main/java/com/puppycrawl/tools/checkstyle/checks/TranslationCheck.java

```diff
@@ -68,9 +72,17 @@
  * <p>
  * app-dev.properties //separator is '-'
  * </p>
+ *
+ * <p><b>checkedLanguageCodes</b> which allows to specify language codes of
+ * translation files which must exist in project. For example, if you need
+ * to check that Japanese and French localization files are present in your project,
+ * you should set checkedLanguageCodes to "ja, fr".
```

Comment:
@mkordas 
Corrected javadoc/xdoc. Added check for default translation.
The name of default translation file begins with the base name of the ResourceBundle and ends with the .properties suffix.
FYI: https://docs.oracle.com/javase/tutorial/i18n/resbundle/propfile.html